daemon/emer-daemon.c

```diff
@@ -959,15 +921,6 @@ has_owner (GDBusProxy *dbus_proxy)
 }
 
 static void
-handle_login_manager_name_owner_set (GDBusProxy *dbus_proxy,
-                                     GParamSpec *pspec,
-                                     EmerDaemon *self)
-{
-  if (has_owner (dbus_proxy))
-    register_with_login_manager (self);
-}
-
-static void
 connect_to_login_manager (EmerDaemon *self)
```

Comment:
We should remove this entire method now as well as removing `login_manager_proxy` from private state.